include/Gaffer/Path.h

```diff
@@ -190,6 +191,10 @@ class GAFFER_API Path : public IECore::RunTimeTyped
 		/// made.
 		virtual const Plug *cancellationSubject() const;
 
+		/// May be implemented by Paths to provide a Context useful for inspecting
+		/// data represented by the Path.
+		virtual const Context *inspectionContext( const IECore::Canceller *canceller = nullptr ) const;
```

Comment:
> This should also be returning ContextPtr rather than Context * since the implementations are creating a new object and passing ownership to the caller.

Cool, swapped over to `ContextPtr` in 5a2b0312, f4bb3527, 9513af1f, and bb583044.